spec/models/miq_report/generator_spec.rb

```diff
@@ -0,0 +1,82 @@
+describe MiqReport::Generator do
+  before do
+    EvmSpecHelper.local_miq_server
+    @user = FactoryGirl.create(:user_with_group)
+    @time_profile_all = FactoryGirl.create(:time_profile_with_rollup, :tz => "UTC")
+    @host1 = FactoryGirl.create(:host)
+  end
+
+  describe "#generate" do
+    context "Memory Utilization Trends report (daily)" do
+      before :each do
+        @miq_report_profile_all = MiqReport.new(
```

Comment:
@yrudman I think so. It's not clear here, but this report is not valid because it doesn't have a name, title, or report group.